config/config2.xsd

```diff
@@ -1180,10 +1180,7 @@
     <xs:element name="profile">
         <xs:complexType>
             <xs:all>
-                <xs:choice minOccurs="1" maxOccurs="1">
-                        <xs:element name="mimetype" type="xs:string"/>
-                        <xs:element ref="mimetype"/>
-                </xs:choice>
+                <xs:element name="mimetype" type="xs:string" minOccurs="1" maxOccurs="1"/>
```

Comment:
The intension was to allow the full subelement or the string here.

I'm not an expert in XSD maybe it needs another nesting.

Reply:
Solved, but there is no optimal solution :(